grpc/tests/grpctest.cpp

```diff
@@ -108,15 +104,14 @@ int main(int /*argc*/, const char * /*argv*/[]) {
   auto stub = MyGame::Example::MonsterStorage::NewStub(channel);
 
 
-  flatbuffers::FlatBufferBuilder fbb;
+  flatbuffers::grpc::MessageBuilder fbb;
   {
     grpc::ClientContext context;
     // Build a request with the name set.
     auto monster_offset = CreateMonster(fbb, 0, 0, 0, fbb.CreateString("Fred"));
     fbb.Finish(monster_offset);
-    auto request = flatbuffers::BufferRef<Monster>(fbb.GetBufferPointer(),
-                                                   fbb.GetSize());
-    flatbuffers::BufferRef<Stat> response;
+    auto request = fbb.ReleaseMessage<Monster>();
+    flatbuffers::grpc::Message<Stat> response;
 
     // The actual RPC.
     auto status = stub->Store(&context, request, &response);
```

Comment:
Unless there's specifically a code added for "parsing error" or somesuch, I think the best we can do is check the error string, which might be a little brittle. But I can certainly add that.

Btw, in case you didn't see, I'll be updating the error code to `INTERNAL` to match the protobuf gRPC behavior when the message is unparseable.